### DISCUSSION TITLE
Hide TOTP Key

### DISCUSSION THREAD

**Discussion by NHAS** [dani-garcia/vaultwarden#2201] [Ideas]
Within the web application the TOTP key is displayed by default with no way to hide it. 

It would be better if this was similar to the password. Default to being hidden but the open to be shown. 
This would reduce the risk of someone photographing the value when I open the web interface in public. 

Example of unhidden TOTP key:
![image](https://user-images.githubusercontent.com/6820641/147945555-68789616-695b-483c-b79a-1f0ae81ba40c.png)

**Comment by BlackDex** [comment]
I suggest to request this upstream: https://community.bitwarden.com/c/feature-requests/5
